Python bindings fail on install

I'm running with `master` and discovered the following build issue with the Python bindings.

```
./configure --prefix=/home/pmixer/tmp-build-CB5NR/install-pmix \
    --with-libevent=/home/pmixer/local/libevent --with-hwloc=/home/pmixer/local/hwloc \
    --enable-python-bindings
make -j 20
make -j 20 install
```

The `make install` will fail with this message:
```
make[2]: Entering directory `/home/pmixer/tmp-build-CB5NR/openpmix/bindings/python'
make[3]: Entering directory `/home/pmixer/tmp-build-CB5NR/openpmix/bindings/python'
/usr/bin/python3 ../../bindings/python/construct.py --src="../../include" --include-dir="../../include"
make[3]: Nothing to be done for `install-data-am'.
BAD DIMENSION 2
make[3]: *** [install-exec-local] Error 1
make[3]: Leaving directory `/home/pmixer/tmp-build-CB5NR/openpmix/bindings/python'
make[2]: *** [install-am] Error 2
make[2]: Leaving directory `/home/pmixer/tmp-build-CB5NR/openpmix/bindings/python'
make[1]: *** [install-recursive] Error 1
make[1]: Leaving directory `/home/pmixer/tmp-build-CB5NR/openpmix/bindings'
make: *** [install-recursive] Error 1
```

--- a/bindings/python/construct.py
+++ b/bindings/python/construct.py
@@ -17,11 +17,11 @@ def harvest_constants(options, src, constants, definitions):
     path = os.path.join(options.src, src)
     # open the file
     try:
-        inputfile = open(path, "r")
+        inputfile = open(path, "r", encoding="utf-8")
     except:
         path = os.path.join(options.includedir, src)
         try:
-            inputfile = open(path, "r")
+            inputfile = open(path, "r", encoding="utf-8")
         except:
             print("File", path, "could not be opened")
             return -1

